### ISSUE TITLE
Install Fail Unable to launch browser for worker, error message: Failed to launch chrome!

### ISSUE THREAD

**Issue by neotokio** [NikolaiT/se-scraper#15]
Installation fails on Debian 9

Some puppeteer issue.

node v10.15.3
npm v6.4.1

This is related to some recent changes? I remember installing se-scraper about 3 weeks ago without any problems.

```
user@host:~/sescrap$ node run.js
[se-scraper] started at [Sun, 24 Mar 2019 18:10:37 GMT] and scrapes google with 2 keywords on 3 pages each.
(node:1932) UnhandledPromiseRejectionWarning: Error: Unable to launch browser for worker, error message: Failed to launch chrome!
[0324/191038.478962:FATAL:zygote_host_impl_linux.cc(116)] No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.
#0 0x562eaff70f29 base::debug::CollectStackTrace()
#1 0x562eafed6593 base::debug::StackTrace::StackTrace()
#2 0x562eafeead1e logging::LogMessage::~LogMessage()
#3 0x562eb155390e service_manager::ZygoteHostImpl::Init()
#4 0x562eafb2c1b7 content::ContentMainRunnerImpl::Initialize()
#5 0x562eafb5efca service_manager::Main()
#6 0x562eafb2a791 content::ContentMain()
#7 0x562eb41c6178 headless::(anonymous namespace)::RunContentMain()
#8 0x562eb41c6205 headless::HeadlessBrowserMain()
#9 0x562eafb5dca3 headless::HeadlessShellMain()
#10 0x562eada951ac ChromeMain
#11 0x7fb4da22c2e1 __libc_start_main
#12 0x562eada9502a _start

Received signal 6
#0 0x562eaff70f29 base::debug::CollectStackTrace()
#1 0x562eafed6593 base::debug::StackTrace::StackTrace()
#2 0x562eaff70ab1 base::debug::(anonymous namespace)::StackDumpSignalHandler()
#3 0x7fb4e03bb0e0 <unknown>
#4 0x7fb4da23efff gsignal
#5 0x7fb4da24042a abort
#6 0x562eaff6f8e5 base::debug::BreakDebugger()
#7 0x562eafeeaf61 logging::LogMessage::~LogMessage()
#8 0x562eb155390e service_manager::ZygoteHostImpl::Init()
#9 0x562eafb2c1b7 content::ContentMainRunnerImpl::Initialize()
#10 0x562eafb5efca service_manager::Main()
#11 0x562eafb2a791 content::ContentMain()
#12 0x562eb41c6178 headless::(anonymous namespace)::RunContentMain()
#13 0x562eb41c6205 headless::HeadlessBrowserMain()
#14 0x562eafb5dca3 headless::HeadlessShellMain()
#15 0x562eada951ac ChromeMain
#16 0x7fb4da22c2e1 __libc_start_main
#17 0x562eada9502a _start
  r8: 0000000000000000  r9: 00007ffdc80a7e20 r10: 0000000000000008 r11: 0000000000000246
 r12: 00007ffdc80a80b8 r13: 0000000000000161 r14: 00007ffdc80a8a20 r15: 00007ffdc80a8a18
  di: 0000000000000002  si: 00007ffdc80a7e20  bp: 00007ffdc80a8060  bx: 0000000000000006
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007fb4da23efff  sp: 00007ffdc80a7e98
  ip: 00007fb4da23efff efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.


TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

    at Cluster.<anonymous> (/home/user/sescrap/node_modules/se-scraper/src/puppeteer-cluster/dist/Cluster.js:143:23)
    at Generator.throw (<anonymous>)
    at rejected (/home/user/sescrap/node_modules/se-scraper/src/puppeteer-cluster/dist/Cluster.js:5:65)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:1932) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:1932) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by NikolaiT** [comment]
Could you please paste `run.js` here?

And did you download chromium via puppeteer?

**Comment by neotokio** [comment]
I used `run.js` from examples given in github Quickstart section, without modifying.

Se-scraper was installed from npm. I also tried building from source. Both fail with same error.

**Comment by NikolaiT** [comment]
Could you fix this?

I installed `se-scraper` on my Ubuntu 18.04 server without any issues. 

I had an similar error when I tried to run the scraper with root privileges. Try to run it with a less privileged user. Did that fix it?

**Comment by deniro21** [comment]
I solved this problem on Debian with:
`sudo sysctl -w kernel.unprivileged_userns_clone=1`
https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox

**Comment by deniro21** [comment]
Also, I would like to point you to a mistake (in my opinion). Having received an error on Debian, I decided to install the original puppeteer and got the same error. After that, I turned off the sandbox and the error disappeared. In the Se-scraper, as described in readme, the sandbox is disabled default. But it seems to me that this is not the case, since the above error simply could not have occurred.
